man/systemd-debug-generator.xml

```diff
@@ -70,6 +70,39 @@
     <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>.</para>
   </refsect1>
 
+  <refsect1>
+    <title>System Credentials</title>
+
+    <variablelist class='system-credentials'>
+      <varlistentry>
+        <term><varname>systemd.unit.*</varname></term>
+        <term><varname>systemd.initrd.unit.*</varname></term>
```

Comment:
i'd avoid this. instead just tell people in the man page to use ConditionFileExists=/etc/initrd-release or so, like we usually do it.